code/modules/surgery/organs/external/_external_organ.dm

```diff
@@ -53,6 +53,15 @@
 	if(restyle_flags)
 		RegisterSignal(src, COMSIG_ATOM_RESTYLE, PROC_REF(on_attempt_feature_restyle))
 
+/obj/item/organ/external/Insert(mob/living/carbon/receiver, special = FALSE, movement_flags)
```

Comment:
Please use the mob_insert() proc, since we only care about the mob here

Reply:
That actually won't work correctly - bodypart_insert() is where bodypart overlays are added, and it's called _after_ mob_insert(). I didn't want to call update_body_parts in bodypart_insert though, since it says not to include owner operations. Hence why I overrode Insert() to do the update as the absolute last thing.

Reply:
Agony